libBuild.sh

```diff
@@ -270,7 +270,7 @@ function publish_docker_ecr {
     language_flag=$(echo "$runtime_name" | sed 's/[0-9].*//')
 
     if [[ ${runtime_name} =~ 'extension' ]];
-    then version_flag=$EXTENSION_VERSION
+    then version_flag=$EXTENSION_VERSION && language_flag="lambdaextension"
```

Comment:
As we are assigning the values in the variable and not checking condition, can we write as below ?

if [[ ${runtime_name} =~ 'extension' ]]; then
  version_flag=$EXTENSION_VERSION
  language_flag="lambdaextension"
fi

Reply:
Updated in the new commit. Thank you!